bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -185,6 +201,10 @@
 	<channel-type id="power-r">
 		<item-type>Number:Power</item-type>
 		<label>Power Readonly DataPoint</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
+		</tags>
 		<state readOnly="true"/>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 		<config-description>
```

Comment:
`flowrate-r` is not tagged. Cannot tell what it is (as it depends on the option set), but for sure `Measurement`.